docs/dev-tools/chopsticks/overview.md

```diff
@@ -269,15 +262,13 @@ These are the methods that can be invoked and their parameters:
 
     === "Parameters"
 
-        |        Name         |                                 Type                                  |                    Description                     |
-        | :-----------------: | :-------------------------------------------------------------------: | :------------------------------------------------: |
-        |       `count`       |                                number                                 |           The number of blocks to build            |
-        |        `dmp`        |                { msg: 0x${string} ; sentAt: number }[]                |   The downward messages to include in the block    |
-        |       `hrmp`        | Record < string \| number, { data: 0x${string} ; sentAt: number }[] > |  The horizontal messages to include in the block   |
-        |        `to`         |                                number                                 |            The block number to build to            |
-        |   `transactions`    |                             0x${string}[]                             |      The transactions to include in the block      |
-        |        `ump`        |                   Record < number, 0x${string}[] >                    |    The upward messages to include in the block     |
-        | `unsafeBlockHeight` |                                number                                 | Build block using a specific block height (unsafe) |
+        - `count` (*number*) - the number of blocks to build
```

Comment:
I think these should be changed to use the type formatting like we do in the XCM SDK.

To implement it, you would use this syntax:

```
- `count` ++"number"++ - the number of blocks to build
```

Then it would look somewhat like this - wdyt?

<img width="533" alt="Screenshot 2024-07-29 at 10 46 46 PM" src="https://github.com/user-attachments/assets/ae28bf12-9a08-4117-95c2-009c0bf1445c">